plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchSmokeTest.java

```diff
@@ -283,6 +283,44 @@ public void testArrayFields()
                 "VALUES ('hello', 20, 30, 60, NULL)");
     }
 
+    @Test
+    public void testMixedArray()
+            throws IOException
+    {
+        String indexName = "test_arrays";
+
+        @Language("JSON")
+        String mapping = "" +
+                "{" +
+                "      \"_meta\": {" +
+                "        \"presto\": {" +
+                "          \"a\": {" +
+                "                \"isArray\": true" +
+                "          }" +
+                "        }" +
+                "      }," +
+                "      \"properties\":{" +
+                "        \"a\": {" +
+                "          \"type\": \"keyword\"" +
+                "        }" +
+                "      }" +
+                "}";
+
+        createIndex(indexName, mapping);
+
+        index(indexName, ImmutableMap.<String, Object>builder()
+                .put("a", "hello")
+                .build());
+
+        index(indexName, ImmutableMap.<String, Object>builder()
```

Comment:
optional: we could add an assertion in the test that missing value is still interpreted as `null` array and not something funny like `array[null]`.